src/Core/View.js

```diff
@@ -278,4 +279,49 @@ View.prototype.getLayers = function getLayers(filter) {
     return result;
 };
 
+/**
+ * @typedef {object} FrameRequester
+ * @property {Function(dt, firtLoop)} update - Method that will be called each
```

Comment:
What I don't like is that it's so easily messed up if controls (for instance) don't plug them well into this system. It relies on FrameRequesters to call `notifyChange` in the same microtask as `MainLoop._step` to work correctly (which GlobeControls does not yet). Otherwise, firstLoop will always be true.

Apart from that, I use it in `FPControls`, so it's at least *a bit* useful...